src/components/RemediationActivityTable.js

```diff
@@ -0,0 +1,109 @@
+    Table,
+    TableHeader,
+    TableBody,
+    expandable
+} from '@patternfly/react-table';
+
+import { DateFormat } from '@redhat-cloud-services/frontend-components';
+
+import { StatusSummary, normalizeStatus } from './statusHelper';
+
+import { PermissionContext } from '../App';
+
+const RemediationActivityTable = ({ remediation, playbookRuns }) => {
+
+    const [ rows, setRows ] = useState([]);
+    const permission = useContext(PermissionContext);
+
+    const generateRows = (playbookRuns) => {
+        return (playbookRuns.reduce((acc, playbooks, i) => (
+            [
+                ...acc,
+                {
+                    isOpen: false,
+                    cells: [
+                        { title: <Link to={ `/${remediation.id}/${playbooks.id}` }><DateFormat type='exact' date={ playbooks.created_at }/></Link>,
+                            cellFormatters: [ expandable ]},
+                        `${playbooks.created_by.first_name} ${playbooks.created_by.last_name}`,
+                        { title: <StatusSummary
+                            executorStatus={ normalizeStatus(playbooks.status) }
+                            counts={ playbooks.executors.reduce((acc, ex) => (
+                                { pending: acc.pending + ex.counts.pending,
+                                    running: acc.running + ex.counts.running,
+                                    success: acc.success + ex.counts.success,
+                                    failure: acc.failure + ex.counts.failure,
+                                    canceled: acc.canceled + ex.counts.canceled
+                                }), { pending: 0, running: 0, success: 0, failure: 0, canceled: 0 }) }
+                            permission={ permission }/>
+                        }
+                    ]
+                }, {
+                    parent: 2 * i,
+                    fullWidth: true,
+                    cells: [{
+                        title: <Table
+                            aria-label="Compact expandable table"
+                            cells={ [ 'Connection', 'Systems', 'Playbook run status' ] }
+                            rows={ playbooks.executors.map(e => (
+                                { cells: [
+                                    { title:
+                                        <Link to={ `/${remediation.id}/${playbooks.id}/${e.executor_id}` }>
+                                            { e.executor_name }
+                                        </Link>
+                                    },
+                                    e.system_count,
+                                    { title: <StatusSummary
+                                        executorStatus={ normalizeStatus(e.status) }
+                                        counts={ e.counts }
+                                        permission={ permission }
+                                        // onCancel={ () => console.log('cancel') } TODO add back
```

Comment:
Cancelation will only be possible at the playbook_run level (you can cancel the entire run but not one particular executor in a running run)